src/calculateRentalCost.js

```diff
@@ -29,6 +29,18 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const sevenDaysBonus = 50;
+  const threeDaysBonus = 20;
+
+  if (days >= 3 && days < 7) {
+    return days * 40 - threeDaysBonus;
+  }
+
+  if (days >= 7) {
+    return days * 40 - sevenDaysBonus;
+  } else {
+    return days * 40;
+  }
```

Comment:
you can implement this part without `else`

Reply:
I agree with illia